src/Symfony/Component/Notifier/Bridge/Slack/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * Check for maximum number of buttons in Slack action block
+ * Add HeaderBlock for slack notifier
```

Comment:
```suggestion
 * Add HeaderBlock
```

Not sure we need it because its the changelog of the slack bridge

Reply:
thanks